README.md

```diff
@@ -31,3 +31,27 @@ Pressing the button will delete the notification for all administrators.
 
 
 After 5 "infected" uploads within 30 minutes, the clients of the user get blocked automatically to prevent further damage to the data. After the problem has been solved, the clients can be re-allowed in the personal settings of the user.
+
+## Configuration settings
+
+Configuration is managed on the **Settings » Security** page under the heading **Ransomware protection**.
+
+You can choose to ignore extensions from the [resources/extensions.txt file](https://github.com/nextcloud/ransomware_protection/blob/master/resources/extensions.txt "Link to the latest version of this file on Github. This may differ from your locally installed version.") by entering the pattern in to the 'Exclude extension patterns' field. You must enter patterns *exactly* as found in the extensions.txt file; entering something that is not a line in that file has no effect.
```

Comment:
```suggestion
You can choose to ignore extensions from the [resources/extensions.txt file](https://github.com/nextcloud/ransomware_protection/blob/master/resources/extensions.txt "Link to the latest version of this file on Github. This may differ from your locally installed version.") by entering the pattern into the 'Exclude extension patterns' field. You must enter patterns *exactly* as found in the extensions.txt file; entering something that is not a line in that file has no effect.
```